src/evo/assetlocktx.cpp

```diff
@@ -140,7 +140,8 @@ bool CAssetUnlockPayload::VerifySig(const llmq::CQuorumManager& qman, const uint
     const auto quorum = qman.GetQuorum(llmqType, quorumHash);
     // quorum must be valid at this point. Let's check and throw error just in case
     if (!quorum) {
-        return state.Invalid(TxValidationResult::TX_CONSENSUS, "internal-error");
+        LogPrint(BCLog::CREDITPOOL, "No quorum for found for hash=%s\n", quorumHash.ToString());
+        return state.Invalid(TxValidationResult::TX_CONSENSUS, "bad-assetunlock-missing-quorum");
```

Comment:
it's indeed an internal error. just 10 lines above it checks, that quorum INDEED exists.
https://github.com/dashpay/dash/pull/6132/commits/5964f0ab06d34228803f8b297b5721cbf8f792c5#diff-496e4543373df8c86c4aecd45188c8dd2267f8d505aea87ae373cbad38c1860eR130

Reply:
Resolved in latest push!